pkg/apis/coordination/validation/validation.go

```diff
@@ -48,5 +55,112 @@ func ValidateLeaseSpec(spec *coordination.LeaseSpec, fldPath *field.Path) field.
 		fld := fldPath.Child("leaseTransitions")
 		allErrs = append(allErrs, field.Invalid(fld, spec.LeaseTransitions, "must be greater than or equal to 0"))
 	}
+	if spec.Strategy != nil {
+		allErrs = append(allErrs, ValidateCoordinatedLeaseStrategy(*spec.Strategy, fldPath.Child("strategy"))...)
+	}
+	// spec.RenewTime is a MicroTime and doesn't need further validation
+	return allErrs
```

Comment:
do you need to validate PreferredHolderID?  

I see that we do not validate holderID at all, so we can't add validation for that, but we could add a warning (a different PR, I think).

Reply:
can I do that in a follow up PR so preferredHolder and holderIdentity validations are in sync?

Reply:
ok